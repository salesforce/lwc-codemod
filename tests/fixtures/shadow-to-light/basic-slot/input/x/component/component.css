slot {
    overflow-y: auto;
}

.foo > slot {
    border: 1px solid black;
}
