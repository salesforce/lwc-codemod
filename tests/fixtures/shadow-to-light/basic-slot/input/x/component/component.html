<template>
    <slot onscroll={onScroll}></slot>
</template>
