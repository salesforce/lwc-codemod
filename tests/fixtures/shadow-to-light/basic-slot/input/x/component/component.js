import { LightningElement } from 'lwc'

export default class extends LightningElement {
  onScroll() {
    console.log('scroll!')
  }
  renderedCallback() {
    console.log('slot', this.template.querySelector('slot'))
  }
}
