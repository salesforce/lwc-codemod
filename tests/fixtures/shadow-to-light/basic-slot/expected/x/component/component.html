<template lwc:render-mode="light">
    <div onscroll={onScroll} class="lwc-slot-wrapper"><slot></slot></div>
</template>
